src/Image/Location.php

```diff
@@ -47,32 +36,24 @@ public function __construct(array $exif)
 
 	/**
 	 * Returns the latitude
-	 *
-	 * @return float|null
 	 */
-	public function lat()
+	public function lat(): float|null
 	{
 		return $this->lat;
 	}
 
 	/**
 	 * Returns the longitude
-	 *
-	 * @return float|null
 	 */
-	public function lng()
+	public function lng(): float|null
 	{
 		return $this->lng;
 	}
 
 	/**
 	 * Converts the gps coordinates
-	 *
-	 * @param string|array $coord
-	 * @param string $hemi
-	 * @return float
 	 */
-	protected function gps($coord, string $hemi): float
+	protected function gps(string|array $coord, string $hemi): float
```

Comment:
`count()` will throw a `TypeError` on string. Not sure how this is supposed to work with the current code.